assets/js/llms-quiz.js

```diff
@@ -117,13 +117,6 @@
 				}
 			} );
 
-			// warn when quiz is running and user tries to leave the page.
```

Comment:
why are we removing this?

Reply:
Sorry, this is corrected. Now the warning will be visible only if the quiz is not resumable.